week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -2,89 +2,99 @@
 // Look at the tests and see how you can fix them.
 
 function mood() {
-  let isHappy = true;
+    let isHappy = true;
+
+    if (isHappy != true) {
+        return "I am happy";
+    } else {
+        return "I am not happy";
+    }
 
-  if (isHappy) {
-    return "I am happy";
-  } else {
-    return "I am not happy";
-  }
 }
+console.log(mood(""));
+//console.log(mood);
 
 function greaterThan10() {
-  let num = 10;
-  let isBigEnough;
-
-  if (isBigEnough) {
-    return "num is greater than or equal to 10";
-  } else {
-    return "num is not big enough";
-  }
+    let num = 10;
+    let isBigEnough = 12;
+
+    if (num <= isBigEnough) {
+        return "num is greater than or equal to 10";
+    } else {
+        return "num is not big enough";
+    }
+
 }
+console.log(greaterThan10(""));
 
 function sortArray() {
-  let letters = ["a", "n", "c", "e", "z", "f"];
-  let sortedLetters;
-
-  return sortedLetters;
+    let letters = ["a", "n", "c", "e", "z", "f"];
+    let sortedLetters = letters;
+    letters.sort();
+    return sortedLetters;
 }
+console.log(sortArray(""));
 
 function first5() {
-  let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
-  let sliced;
+    let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
+    let sliced = numbers.slice(0, 5);
+    //numbers = numbers.sliced(0, 4);
 
-  return sliced;
+    return sliced;
 }
+console.log(first5(""));
 
 function get3rdIndex(arr) {
-  let index = 3;
-  let element;
+    //let ary = ["fruit", "banana", "apple", "strawberry", "raspberry"];
+    let index = ["fruit", "banana", "apple", "strawberry", "raspberry"];
+    let index = [11, 37, 62, 18, 19, 3, 30];
+    let element = index[3];
```

Comment:
- you are passing an array as an argument , you don't need any statement inside .
```javascript
function get3rdIndex(arr) {
return arr[2]
}
```
- `arr[2]` because we start from 0.
- when you call the function you pass an array:
```javascript
function get3rdIndex([1,2.3,4.5,6]);
```